src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -597,28 +627,28 @@ <h2 class="m1">Apple</h2>
-    <div>
-      <amp-accordion>
-        <section>
-          <h4 class="p1">Description</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Specification</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Measurements</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Other</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-      </amp-accordion>
-    </div>
+    <!-- ## Tab panels -->
+    <!-- Use `amp-selector` styled as tab panels to add additional data about your product. Learn more about [`amp-selector`](https://www.ampproject.org/docs/reference/components/dynamic/amp-selector). -->
+  <amp-selector role="tablist"
+  layout="container"
+  class="ampTabContainer p1">
+  <div role="tab"
+    class="tabButton h4"
+    selected
+    option="a">ABOUT</div>
+  <div role="tabpanel"
+      class="tabContent p1 p">Fruit is rich in vitamins and minerals. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
+  <div role="tab"
+    class="tabButton h4"
+    option="b">SPECS</div>
+  <div role="tabpanel"
+    class="tabContent p1 p">Always wash fruit before eating. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+  <div role="tab"
+      class="tabButton h4"
+      option="c">MEASUREMENTS</div>
+  <div role="tabpanel"
+      class="tabContent p1 p">Size may vary. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</div>
```

Comment:
s/p1/py1 as the left and right indentation does not fit with the rest of the sample.